_posts/en/newsletters/2023-08-02-newsletter.md

```diff
@@ -46,6 +46,93 @@ Bitcoin infrastructure software.
     find the thread interesting for its discussion of cryptographic
     techniques. {% assign timestamp="5:07" %}
 
+## Bitcoin Core PR Review Club
```

Comment:
> @LarryRuane this went into last weeks newsletter (262) instead of this weeks (263)

Oh, I guess we're not doing time travel yet ... sorry about that, fixed.